shared/src/Datadog.Trace.ClrProfiler.Native/util.h

```diff
@@ -51,6 +52,8 @@ const shared::WSTRING default_exclude_assemblies[]{
     WStr("VBCSCompiler.exe"),
     WStr("vsdbg"),
     WStr("vsdbg.exe"),
+    WStr("WaAppAgent.exe"),                             // https://github.com/Azure/WindowsVMAgent
+    WStr("WindowsAzureGuestAgent.exe")                  // https://github.com/Azure/WindowsVMAgent
```

Comment:
So tbh, the casing thing is interesting. In general I think we kept the casing because it increases the specificity, especially as we _thought_ these would always have the standard casing, and it avoided any potential clashes with customer apps that should be traced.

Any way, if we're going to do case-insensitive comparison, we should just pre-transform these to the correct case I think.

Reply:
Or maybe it should be case-sensitive on Linux but not on Windows? (because that's how these OSes generally work)

Reply:
Maybe a dumb question but would it matter to be case-_in_sensitive on Linux?

Reply:
I think the only concern is (from @andrewlock above):

> it avoided any potential clashes with customer apps that should be traced.